apollo-gradle-plugin/src/main/kotlin/com/apollographql/apollo/gradle/api/CompilerParams.kt

```diff
@@ -154,4 +154,10 @@ interface CompilerParams {
    * Default value: false
    */
   val generateAsInternal: Property<Boolean>
+
+  /**
+   * Whether to generate GraphQL enum types by matching as Kotlin sealed classes if they match one of provided regex patterns.
+   * By default any GraphQL enum type is generated as `enum class`.
+   */
```

Comment:
```suggestion
  /**
   * A list of [Regex] patterns for GraphQL enums that should be generated as Kotlin sealed classes instead
   * of the default Kotlin enums.
   * Use this if you want your client to have access to the rawValue of the enum. This can be useful if new
   * GraphQL enums are added but the client was compiled against an older schema that doesn't have knowledge
   * of the new enums.
   */
```